interfaces/AW-App/src/app/validation-components/download-data/download-data.component.ts

```diff
@@ -21,8 +21,11 @@ class ValidationAnswer {
 })
 export class DownloadDataComponent implements ValidationComponent {
   public downloadReady = false;
+  public downloadAborted = false;
   public nrDownloaded: number;
 
+  public validationData: ValidationAnswer[];
```

Comment:
Properties or members of a class only need to be declared `public` if they will be used from 'outside' the class. (So most times only when used in the template)

If that is not the case (as it is here), you should set these properties to `private`.

Reply:
Good point. I know this, but my default-mode-without-thinking is still to set 'public'. I will improve!